### PR TITLE
make set_max_expansion configurable for Xapian

### PR DESCRIPTION
**Pull request by kfa-wtag**
https://garaiorem.atlassian.net/browse/REM-24399

**Description:**

ACs:
1. set_max_expansion is configurable over the xapian_db.yml
2. If we configure set_max_expansion it is correctly used in the GEM
3. if we don’t have the set_max_expansion option set in the xapian_db.yml it should not set set_max_expansion

### CODE REVIEW

File: lib/xapian_db/railtie.rb [hunk 1]
```diff
@@ -85,6 +86,7 @@ def self.configure_from(env_config)
       @term_min_length      = env_config["term_min_length"]
       @enable_phrase_search = env_config["enable_phrase_search"] == true
       @term_splitter_count  = env_config["term_splitter_count"] || 0
+      @set_max_expansion = env_config["set_max_expansion"]
```
Comment (ssa-wtag) [inline lib/xapian_db/railtie.rb:89]:
```suggestion
      @set_max_expansion     = env_config["set_max_expansion"]
```

**Review by ssa-wtag — COMMENTED** [review]
I think we have got it covered except the following?

`if we don’t have the set_max_expansion option set in the xapian_db.yml it should not set set_max_expansion`

**Comment by kfa-wtag** [comment]
> I think we have got it covered except the following?
> 
> `if we don’t have the set_max_expansion option set in the xapian_db.yml it should not set set_max_expansion`

cool, I have covered the acceptance point here `lib/xapian_db/query_parser.rb`. Thanks!

**Comment by gernotkogler** [comment]
Thanks for your help guys, release 1.3.12 is on rubygems

**Pull request merged**